package.json

```diff
@@ -27,81 +27,81 @@
     "@tableflip/react-dropdown": "^1.1.1",
     "@tableflip/react-inspector": "^2.3.0",
     "brace": "^0.11.1",
-    "chart.js": "^2.7.2",
+    "chart.js": "^2.7.3",
     "d3": "^5.7.0",
     "details-polyfill": "^1.1.0",
     "file-extension": "^4.0.5",
     "filesize": "^3.6.1",
-    "i18next": "^11.6.0",
-    "i18next-browser-languagedetector": "^2.2.3",
+    "i18next": "^11.10.2",
+    "i18next-browser-languagedetector": "^2.2.4",
     "i18next-icu": "^0.6.0",
     "i18next-xhr-backend": "^1.5.1",
-    "internal-nav-helper": "^1.0.2",
+    "internal-nav-helper": "^1.1.0",
     "ipfs-css": "^0.10.0",
     "ipfs-geoip": "^2.3.0",
-    "ipfs-redux-bundle": "^4.1.1",
-    "ipfs-unixfs": "^0.1.15",
-    "ipld": "^0.17.3",
+    "ipfs-redux-bundle": "^4.1.2",
+    "ipfs-unixfs": "^0.1.16",
+    "ipld": "^0.20.1",
     "ipld-explorer-components": "^1.0.0",
     "is-binary": "^0.1.0",
-    "is-ipfs": "^0.4.2",
+    "is-ipfs": "^0.4.8",
     "milliseconds": "^1.0.3",
-    "money-clip": "^2.1.0",
-    "multiaddr": "^5.0.2",
+    "money-clip": "^3.0.0",
+    "multiaddr": "^6.0.0",
     "multiaddr-to-uri": "^4.0.0",
     "prop-types": "^15.6.2",
     "pull-file-reader": "^1.0.2",
-    "react": "^16.4.2",
-    "react-ace": "^6.1.4",
+    "react": "^16.6.3",
+    "react-ace": "^6.2.0",
     "react-chartjs-2": "^2.7.4",
     "react-copy-to-clipboard": "^5.0.1",
     "react-country-flag": "^1.0.1",
-    "react-debounce-render": "^4.0.3",
-    "react-dnd": "^5.0.0",
-    "react-dnd-html5-backend": "^5.0.1",
-    "react-dom": "^16.4.2",
+    "react-debounce-render": "^5.0.0",
+    "react-dnd": "^7.0.2",
```

Comment:
https://github.com/react-dnd/react-dnd/releases
Need to check the drag and drop to add files on the files page, we lack a test for that